presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -298,6 +298,20 @@ public void testTableProperties()
         computeActual("CREATE TABLE test_table_properties_2 (foo BIGINT, bar BIGINT, ds DATE) WITH (ORDERING=array['foo','bar'], TEMPORAL_COLUMN='ds')");
     }
 
+    @Test
+    public void testSystemTableInQuery()
```

Comment:
This is not a great method name, since it is specifically testing the `system.shards` table, but the method name indicates that it is a generic test for all system tables. I move this test query to be in `testShardsSystemTable`, since they're both testing the same table.